test/e2e/commands/test_ingest_xmlupload.py

```diff
@@ -0,0 +1,52 @@
+import shutil
+from pathlib import Path
+
+import pytest
+
+from dsp_tools.commands.ingest_xmlupload.upload_xml import ingest_xmlupload
+from dsp_tools.models.exceptions import InputError
+
+
+@pytest.fixture()
+def _retrieve_mapping_file() -> None:  # type: ignore[misc]
```

Comment:
if you import Iterator from typing, then you don't need the ignore comment:

```suggestion
def _retrieve_mapping_file() -> Iterator[None]:
```